tokenizers/src/manager.rs

```diff
@@ -54,6 +75,7 @@ pub enum SearchTokenizer {
     ChineseLindera,
     JapaneseLindera,
     KoreanLindera,
+    #[strum(serialize = "icu")]
     #[cfg(feature = "icu")]
```

Comment:
Does this ordering of `#[strum]` then `#[cfg]` still compile if the "icu" feature is not turned on?

Reply:
It still works. Let's reorder the attribute to avoid this confusion.